### PR TITLE
add clarity to -p option

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley somalley@redhat.com

### CODE REVIEW

File: man/docker-run.1.md [hunk 1]
```diff
@@ -353,10 +353,13 @@ ports and the exposed ports, use `docker port`.
...
 **-p**, **--publish**=[]
    Publish a container's port, or range of ports, to the host.
-                               format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
-                               Both hostPort and containerPort can be specified as a range of ports. 
-                               When specifying ranges for both, the number of container ports in the range must match the number of host ports in the range. (e.g., `-p 1234-1236:1234-1236/tcp`)
-                               (use 'docker port' to see the actual mapping)
+
+   Format: `ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort`
+Both hostPort and containerPort can be specified as a range of ports. 
+(e.g. `docker run -p 127.0.0.1:$HOSTPORT:$CONTAINERPORT --name CONTAINER -t someimage`)
```
Comment (thaJeztah) [inline man/docker-run.1.md]: just a nit: we standardize on adding a comma after "e.g."; so `e.g., `

Comment (thaJeztah) [inline man/docker-run.1.md]: Reading; this example comes directly after "can be specified as a range of ports", but the example itself doesn't show a range.

I think it's good to keep an example in to show how to specify a range, w.d.y.t.?

**Comment by moxiegirl** [comment]
Thanks @sallyom LGTM

File: man/docker-run.1.md (hunk 1 above)
Comment (thaJeztah) [inline man/docker-run.1.md]: oh (sorry for spamming); if you add that example, could you use different ranges for "host" and "container" to illustrate they don't have to match, as long as the size of the range is equal?

Comment (sallyom) [inline man/docker-run.1.md]: good to know about the comma :) I'll fix up the example...

**Comment by sallyom** [comment]
@thaJeztah fixed up the examples and added ip example (too much?), good?  thanks for fb.

**Comment by thaJeztah** [comment]
thanks! LGTM

ping @moxiegirl, wanna have a second look at this?

**Comment by moxiegirl** [comment]
Thanks @sallyom @thaJeztah Let's merge it!

**Comment by moxiegirl** [comment]
cherrypick please

**Pull request merged**